src/main/java/de/azapps/kafkabackup/common/record/RecordJSONSerde.java

```diff
@@ -0,0 +1,85 @@
+package de.azapps.kafkabackup.common.record;
+
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+import java.util.Base64;
+import java.io.*;
+
+public class RecordJSONSerde {
+    private ObjectMapper mapper;
```

Comment:
missing `static` ? 🤔

Reply:
auto-reply: no :D if we'll create only one instance of this "serde", then static is not required

Reply:
Yeah, see above. I read this and got scared: https://stackoverflow.com/a/36162525
So you read and write are not static any more. You need to instantiate the Serde.
Is that bad?

Reply:
Oh, and I'm not super-worried about GC, since we can create a RecordJSONSerde in every instance of PartitionWriter and re-use it for the whole lifetime of the partition assignment.

Reply:
I was reading this discussion on StackOverflow, but then I found this: https://www.stubbornjava.com/posts/practical-jackson-objectmapper-configuration :D but the current solution is really okay for me 👍

Reply:
Ok. Yeah. We can maybe improve on it later. At least we are not doing this:
> "The anti-pattern is creating a new ObjectMapper every serialization request (Don't do that!)."
:D